<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDropper</class>
 <widget class="QDialog" name="FileDropper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FiiO File Dropper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="pushItemDown">
     <property name="text">
      <string>&amp;Down</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushRoot">
     <property name="text">
      <string>Get Folder</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="labLength">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>0:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineRoot"/>
   </item>
   <item row="0" column="0" rowspan="5" colspan="3">
    <widget class="QTreeView" name="treeView">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushPause">
     <property name="text">
      <string>Get File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushItemDelete">
     <property name="text">
      <string>De&amp;lete</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="linePauseAudio"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushItemUp">
     <property name="text">
      <string>&amp;Up</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Root Foldder:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pause Audio:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Compilation Time:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushLoadM3u">
     <property name="text">
      <string>L&amp;oad</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushLoadM3u</tabstop>
  <tabstop>pushSave</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushRoot</tabstop>
  <tabstop>pushPause</tabstop>
  <tabstop>pushItemUp</tabstop>
  <tabstop>pushItemDown</tabstop>
  <tabstop>pushItemDelete</tabstop>
  <tabstop>pushClear</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>lineRoot</tabstop>
  <tabstop>linePauseAudio</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FileDropper</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
